extensions/podman/package.json

```diff
@@ -185,6 +185,25 @@
         {
           "id": "podmanInstalled",
           "title": "Podman successfully installed",
+          "when": "onboardingContext:podmanIsNotInstalled == false"
+        },
+        {
+          "id": "preCreatePodmanMachine",
+          "title": "We could not find any Podman machine. Let's create one!",
+          "when": "onboardingContext:podmanMachineExists == false && onboardingContext:isLinux == false"
+        },
+        {
+          "id": "createPodmanMachine",
+          "title": "Create a Podman machine",
+          "when": "onboardingContext:podmanMachineExists == false && onboardingContext:isLinux == false",
+          "completionEvents": [
+            "onboardingContext:podmanMachineExists == true"
+          ],
+          "component": "create"
```

Comment:
Hello, AFAIK I would say we can't go with a simple generic keyword like `create` as it means tons of things. I think we need to be more specific.

A provider can register both kubernetes connection provider and container connection provider (and it could be more in the future)

so probably we should say what we would create

It seems you want to hook it to `setContainerProviderConnectionFactory` and  `setKubernetesProviderConnectionFactory` for a provider

so it should contains at minimal `ContainerProviderConnection` if you want the keyword create then it should be `createContainerProviderConnection` (for creating a podman machine)

it could include Wizard, Page, etc suffix if it could help users to better understand what will be displayed